src/genai/extensions/langchain/llm.py

```diff
@@ -156,6 +161,17 @@ def _generate(
 
         return result
 
+    async def _agenerate(
+        self,
+        prompts: List[str],
+        stop: Optional[List[str]] = None,
+        run_manager: Optional[AsyncCallbackManagerForLLMRun] = None,
+        **kwargs: Any,
+    ) -> LLMResult:
+        return await asyncio.get_running_loop().run_in_executor(
+            None, partial(self._generate, prompts, stop, run_manager, **kwargs)
+        )
```

Comment:
This part has been updated in a newer version of LangChain, but we want to support the old behaviour.